<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDEventGeneralDialog</class>
 <widget class="QDialog" name="CMDEventGeneralDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="typeEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="objectEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="objectLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>430</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>对象</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectObjectButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableLogCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>记录到日志</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
